.github/CODEOWNERS

```diff
@@ -1,3 +1,5 @@
 * @powerhome/heroes-for-hire
 packages/**/*.gemspec
 packages/**/Gemfile
+
+packages/scim_pickens/ @powerhome/s-h-i-e-l-d
```

Comment:
@benlangfeld @wadewinningham is it possible to get the shield team added to this repository?

Reply:
I don't think it is necessarily appropriate to start allocating shared ownership of power-tools.

What are you trying to achieve with this library?